src/components/Thread.vue

```diff
@@ -50,26 +51,23 @@ export default {
 			message: undefined,
 			errorMessage: '',
 			error: undefined,
+			thread: [],
 			expandedThreads: [],
 		}
 	},
 	computed: {
-		thread() {
-			return this.$store.getters.getEnvelopeThread(this.$route.params.threadId)
-		},
 		threadParticipants() {
 			const recipients = this.thread.flatMap(envelope => {
 				return envelope.from.concat(envelope.to).concat(envelope.cc)
 			})
 			return uniqBy(prop('email'), recipients)
 		},
 		threadSubject() {
-			const thread = this.thread
```

Comment:
unrelated change?

Reply:
yes, unrelated change. 
just, I think this line doesn't serve any purpose...?